src/core/stream/index.ts

```diff
@@ -398,14 +402,39 @@ export default function Stream({
       return Observable.of(evt);
     });
 
-    /**
-     * Add management of events linked to live Playback.
-     * @type {Observable}
+    /* In the case where minimumUpdatePeriod is specified, and above 0,
+     * manifest may be updated when update period has elapsed from the download
+     * time of previous manifest.
+     * @param {number} minimumUpdatePeriod
+     * @param {number} updateGap
      */
-    const buffers$ = (manifest.isLive ?
-      handledBuffers$
-        .mergeMap(liveEventsHandler(videoElement, manifest, fetchManifest)) :
-      handledBuffers$);
+    function refreshExpiredManifest(
+      updateGap: number,
+      minimumUpdatePeriod?: number
+    ): Observable<IManifestUpdateEvent> {
+      return minimumUpdatePeriod ?
+        Observable.timer((minimumUpdatePeriod - updateGap) * 1000)
+          .mergeMap(() => {
+            return refreshManifest(fetchManifest, manifest)
+              .concatMap(() =>
```

Comment:
With a concatMap like that, we'll never have the "manifestUpdate" event

Reply:
Also, this is a new requests X seconds after the previous one answers? Not a new request every X seconds?

Reply:
It is actually new request X seconds after the last manifest was loaded.